src/API/Configurations.js

```diff
@@ -0,0 +1,90 @@
+import { useQueryClient } from 'react-query';
```

Comment:
why do your files start from capital? are they classes or components?

Reply:
it's a composite module, with several exports

Reply:
usually it follows camelCase, PascalCase is used by classes or components

Reply:
I guess there's no strict rule for modules.

As a general rule we use camelCase for something that is insnance and can be multiplied,
and PascalCase for something that can be instantiated OR is used as namespace for static members, like in
```js
import React from 'react';

class Abc extends React.Component {
```

They also use the same convention in MDN for module names, see
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#creating_a_module_object

As for naming the files, in React world we use same case for filename as for it content - same PascalCase for components and classes. So I used to name such modules in PascalCase too.

But again, no strict rules for that, it's an opinionated thing.
So if you point me to the rule on this here in FOLIO, I'll gladly abide it.

Reply:
Here is elaborate article on the subject https://badgerbadgerbadgerbadger.medium.com/letter-casing-in-names-of-imported-nodejs-modules-707194c6a003